Nfiann-update (#7193)
fishtown-analytics/docs.getdbt.com: website/snippets/_cloud-license-types.md

There are three license types in dbt Cloud:

- **Developer** &mdash; User can be granted _any_ permissions.

> - **Read-Only** &mdash; Available on Enterprise and Team plans only. User has read-only permissions applied to all dbt Cloud resources. Intended to view the [artifacts](/docs/deploy/artifacts) created in a dbt Cloud account, as well as view the [deploy](/docs/deploy/deployments) section (like jobs, runs, and schedules) in a dbt Cloud account. _Read-only licensed users do not inherit rights from any permission sets_. Every read-only licensed user has the same access across the account, regardless of the group permissions assigned.

Reviewer: ```suggestion
- **Read-Only** &mdash; Available on Enterprise and Team plans only. User has read-only permissions applied to all dbt Cloud resources. Intended to view the [artifacts](/docs/deploy/artifacts) and the [deploy](/docs/deploy/deployments) section (jobs, runs, schedules) in a dbt Cloud account, but can’t make changes. _Read-only licensed users do not inherit rights from any permission sets_. Every read-only licensed user has the same access across the account, regardless of the group permissions assigned.
```
